features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
+    In order to add or remove provinces in existing countries
+    As an Administrator
+    I want to be able to edit a country and its provinces
+
+    Background:
+        Given I am logged in as an administrator
```

Comment:
This steps should be swapped. `I am logged in as an administrator` should be always written as a last one.